modules/user/user.resolver.js

```diff
@@ -1,20 +1,31 @@
 const userService = require('./user.service');
 
+const Unauthorized = `${[
```

Comment:
Please, use the same format of naming as for the `*_NOT_FOUND` error messages

Reply:
fixed